CHANGELOG.md

```diff
@@ -1,10 +1,14 @@
 # Changelog
 
-All notable changes to the Sample Plug-in for Zowe CLI will be documented in this file.
+All notable changes to the Sample plug-in for Zowe CLI will be documented in this file.
```

Comment:
is `Sample` a name? 
if so, I agree with the caps (and we should probably capitalize it in other places)
if not, maybe we should change it to lowercase 😋

Reply:
lowercase! ty